tensorlayer/layers/core.py

```diff
@@ -17,6 +17,22 @@
 
 _global_layer_name_dict = {}  # TODO: better implementation?
 
+_act_dict = {
+    "relu": tf.nn.relu,
+    "relu6": tf.nn.relu6,
+    "leaky_relu": tf.nn.leaky_relu,
+    "lrelu0.2": tf.nn.leaky_relu,
```

Comment:
this is incorrect ..  need to parse the number from `lrelu222.222: lambda x : tf.nn.leaky_relu(x, 222.222)

Reply:
got it

Reply:
@zsdonghao fixed